dockerfiles/image.sh

```diff
@@ -69,6 +69,25 @@ finalize_platform_core_image() {
     docker commit $CONTAINER hpccsystems/platform-core:$GIT_BRANCH-$MODE-$crc
     docker stop $CONTAINER
     docker rm $CONTAINER
+
+    if [ "$MODE" = "debug" ]; then
+        # Add sources
+        echo "--- Adding sources to '$image_name' image ---"
+        local CONTAINER=$(docker run -d \
+            --mount source=$ROOT_DIR,target=/hpcc-dev/HPCC-Platform-local,type=bind,readonly \
+            --mount source=$HPCC_BUILD,target=/hpcc-dev/build,type=volume \
+            hpccsystems/platform-core:$GIT_BRANCH-$MODE-$crc "tail -f /dev/null")
+        docker exec --user root --workdir /hpcc-dev/HPCC-Platform $CONTAINER /bin/bash -c "chown -R hpcc:hpcc /hpcc-dev/HPCC-Platform"
+        docker exec --workdir /hpcc-dev/HPCC-Platform $CONTAINER /bin/bash -c "git config --global --add safe.directory /hpcc-dev/HPCC-Platform-local"
+        docker exec --workdir /hpcc-dev/HPCC-Platform $CONTAINER /bin/bash -c "cp -r ../HPCC-Platform-local/.git ./.git"
+        docker exec --workdir /hpcc-dev/HPCC-Platform $CONTAINER /bin/bash -c "git reset --hard --recurse-submodules"
+        docker exec --workdir /hpcc-dev/HPCC-Platform-local $CONTAINER /bin/bash -c "git ls-files --modified --exclude-standard -z | xargs -0 -I {} cp {} /hpcc-dev/HPCC-Platform/"
```

Comment:
this needs to be:
... cp {} /hpcc-dev/HPCC-Platform/{}

(it's currently copying changed files in to the roo of /hpcc-dev/HPCC-Platform)